tests/unit_tests/test_wave_06.py

```diff
@@ -227,11 +218,8 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert result is False
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_d and item_e and item_f in jesse.inventory
+    assert item_a and item_b and item_c in tai.inventory
```

Comment:
These asserts have the right idea, but there's a subtle bug in them. Take a look at this snippet and see if you can figure out how this same unintuitive behavior could cause problems in your code:
```
data = [1, 2, 3]
a = 1
b = 99
print(b and a in data)
# Prints True
```